build.sbt

```diff
@@ -10,17 +10,18 @@ crossScalaVersions := Seq("2.10.5", "2.11.7")
 
 spName := "databricks/spark-avro"
 
-sparkVersion := "1.3.0"
-
-sparkComponents += "sql"
+sparkVersion := "1.4.0"
 
 spAppendScalaVersion := true
 
 spIncludeMaven := true
 
-libraryDependencies += "org.apache.avro" % "avro" % "1.7.6" exclude("org.mortbay.jetty", "servlet-api")
+sparkComponents := Seq("sql")
 
-libraryDependencies += "org.apache.avro" % "avro-mapred" % "1.7.6" % "provided" exclude("org.mortbay.jetty", "servlet-api")
```

Comment:
Why did you change this dependency to be non-provided? This change is likely to cause Hadoop version mismatch problems for consumers of this library unless we publish different artifacts / POMs for different Hadoop versions. However, I don't think that we need to do that because presumably the existing 1.0.0 release of `spark-avro` works on multiple Hadoop versions.